### DISCUSSION TITLE
Remote Access With Cloudflared

### DISCUSSION THREAD

**Discussion by pstark556** [photoprism/photoprism#2514] [Q&A]
Is there a guide to setup remote access to PhotoPrism with Cloudflared? I tried and can only access it http. If I change it to https I get a 503 error.

**Comment by centralhardware** [comment]
from local network?

**Reply by pstark556** [reply]
In Cloudflare, if I set service to http, I can access it remotely (not on the same network) but it's not secure. Once I change the service to https, I will get a 502 (not 503) error when using local and external networks.

**Reply by ideologysec** [reply]
It doesn't matter if you point cloudflare to the http service. It's doing the securing for you. You can check the HTTPS status in the browser.

**Comment by tsvetkovv** [comment]
@pstark556 There are a lot of quirks with it. Where do you change it? So little details from your side in order to help you.
Mine is like this
```yaml
ingress:
  - hostname: my.domain.com
    service: http://localhost:2342
```
There is also an option to set up everything in the dashboard.
P.S. the issue is completely unrelated to photoprism.

**Comment by Sanookmakmak** [comment]
I have configured the app listening on http://photoprism.lan:2342 and let the reverse proxy from outside of the LAN pointing to it. Cloudflare reverse proxy is working well with that configuration.
